utility/src/main/java/org/oppia/util/parser/StringUtils.kt

```diff
@@ -1,8 +1,11 @@
 package org.oppia.util.parser
 
-/** StringUtils to handle input strings. */
+/** StringUtils to handle html tags strings. */
 object StringUtils {
+  // constant for handling Ordered tag <ol>
```

Comment:
Please provide more details here than just 'constant'. How is this constant used? What does it represent? When should callers use it? The documentation for these constants should try to answer these questions to make it clear how these variables should be used.

Reply:
Added detailed kdoc for the class.